packages/core/components/MenuBar/styles.module.css

```diff
@@ -0,0 +1,46 @@
+.MenuBar {
+  background-color: var(--puck-color-white);
+  border-bottom: 1px solid var(--puck-color-grey-8);
+  display: none;
+  left: 0;
+  margin-top: 1px;
+  padding: 8px 16px;
+  position: absolute;
+  right: 0;
+  top: 100%;
+  z-index: 1;
+}
```

Comment:
It's possible to render the overlay actions on top of this. Can probably be fixed in post.

![image](https://github.com/measuredco/puck/assets/985961/4340963b-819f-4eb7-87e8-240e3e5b30ad)

Reply:
Fixed